randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1215,7 +1225,20 @@ Extra - asset_id: collision_camera_010
   > Mid-Left Spider Magnet Pit
       Trivial
   > Upper Water Ledge - Right
-      Horizontal Water Movement
+      Any of the following:
+          Horizontal Water Movement
+          All of the following:
+              Gravity Suit and Spin Boost
+              Any of the following:
+                  All of the following:
+                      # https://youtu.be/EKucxlQ6XtE
+                      Slide and Slide Jump (Intermediate)
+                  All of the following:
+                      # https://youtu.be/rb01nTDYOXk
+                      Movement (Intermediate)
+                  All of the following:
+                      # https://youtu.be/wlCOhr95pAY
+                      Movement (Beginner) and Walljump (Beginner)
```

Comment:
i personally dont think the wall jump trick here is necessary. or since the difficulty is from performing the wall jump, maybe the movement shouldnt be here. I just dont think it should require both.

Reply:
I removed movement from this and increased the walljump to intermediate.